http_logs/challenges/default.json

```diff
@@ -136,6 +136,20 @@
           "warmup-iterations": 200,
           "iterations": 100,
           "target-throughput": 2
+        },
+        {
+          "name": "desc-sort-with-after-timestamp-after-force-merge-1-seg",
+          "operation": "desc_sort_with_after_timestamp",
+          "warmup-iterations": 200,
+          "iterations": 100,
+          "target-throughput": 2
```

Comment:
the throughput to start should be much smaller since the query is currently very slow.

Reply:
decreased `target-throughput` to `0.5` to make `service time` close to `latency`. Addressed in b9f095f